### PR TITLE
Adds helm chart

### PR DESCRIPTION
**Pull request by marcinc**
Addresses #238

### CODE REVIEW

**Review by gambol99 — DISMISSED** [review]

File: charts/krane/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
```
Comment (gambol99) [inline charts/krane/Chart.yaml:1]: Can if done in another PR ... but might want to consider adding a linter to the helm chart

```yaml
   helm:
      runs-on: ubuntu-latest
      steps:
        - name: Checkout
          uses: actions/checkout@v3
          with:
            fetch-depth: 0
        - name: Set up Helm
          uses: azure/setup-helm@v1
          with:
            version: v3.8.1
        - uses: actions/setup-python@v2
          with:
            python-version: 3.7
        - name: Set up chart-testing
          uses: helm/chart-testing-action@v2.2.1
        - name: Run chart-testing (lint)
         if: github.ref == 'refs/heads/master'
          run: ct lint
        - name: Run chart-testing (lint)
         if: github.ref != 'refs/heads/master'
          run: ct lint --check-version-increment=false
```

It will require you add maintainer links to the chart btw

**Review by gambol99 — APPROVED** [review]

**Review by KashifSaadat — APPROVED** [review]

File: charts/krane/Chart.yaml (hunk 1 above)
Comment (marcinc) [reply charts/krane/Chart.yaml:1]: Good shout. Added in this PR.

**Review by KashifSaadat — DISMISSED** [review]

**Review by gambol99 — APPROVED** [review]
